plugins/uartdmx/README.md

```diff
@@ -12,10 +12,10 @@ http://eastertrail.blogspot.co.uk/2014/04/command-and-control-ii.html
 
 ## Config file: `ola-uartdmx.conf`
 
-`enabled = true`
+`enabled = true`  
```

Comment:
I thought that too so initially went and removed them from everything and then found it rendered oddly as it became an inline code block. Maybe if we changed it to a triple backtick it might work possibly?

Reply:
Mh, I also cannot figure why that would render strange. If adding white space fixes it, just keep that